packages/grafana-ui/src/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -1,53 +1,27 @@
 import { css, cx } from '@emotion/css';
-import React, { useEffect, useRef, useState } from 'react';
-import { useForm } from 'react-hook-form';
+import React from 'react';
 
-import { GrafanaTheme2 } from '@grafana/data';
-import { selectors } from '@grafana/e2e-selectors';
+import { IconName } from '@grafana/data';
 
 import { useStyles2 } from '../../themes';
-import { IconName } from '../../types/icon';
-import { Button, ButtonVariant } from '../Button';
-import { Input } from '../Input/Input';
-import { Box } from '../Layout/Box/Box';
-import { Stack } from '../Layout/Stack/Stack';
+import { ButtonVariant } from '../Button';
 import { Modal } from '../Modal/Modal';
 
-export interface ConfirmModalProps {
+import { ConfirmContent, ConfirmContentProps } from './ConfirmContent';
+
+export interface ConfirmModalProps extends Omit<ConfirmContentProps, 'fillToConfirmText'> {
```

Comment:
I wouldn't extend the props interface here - i think it's fine to just redefine them :)